nifi-commons/nifi-security-utils/src/test/groovy/org/apache/nifi/security/util/KeyStoreUtilsGroovyTest.groovy

```diff
@@ -29,17 +31,22 @@ import org.slf4j.LoggerFactory
 import javax.net.ssl.HttpsURLConnection
 import javax.net.ssl.SSLSocket
 import javax.net.ssl.SSLSocketFactory
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.Paths
 import java.security.KeyStore
 import java.security.cert.Certificate
+import java.security.cert.X509Certificate
 
 @RunWith(JUnit4.class)
 class KeyStoreUtilsGroovyTest extends GroovyTestCase {
     private static final Logger logger = LoggerFactory.getLogger(KeyStoreUtilsGroovyTest.class)
 
-    private static final File KEYSTORE_FILE = new File("src/test/resources/keystore.jks")
-    private static final String KEYSTORE_PASSWORD = "passwordpassword"
-    private static final String KEY_PASSWORD = "keypassword"
-    private static final KeystoreType KEYSTORE_TYPE = KeystoreType.JKS
+    private static final String TEST_KEYSTORE_PASSWORD = KeyStoreUtils.generatePassword()
+    private static final String TEST_KEY_PASSWORD = KeyStoreUtils.generatePassword()
+    private static final String TEST_TRUSTSTORE_PASSWORD = KeyStoreUtils.generatePassword()
+    private static final KeystoreType JKS_STORE_TYPE = KeystoreType.JKS
```

Comment:
Perhaps renaming this variable to something like `DEFAULT_STORE_TYPE` would clarify the reason for declaring it here as opposed to just using the enum value where necessary.